make/check-node-version.js

```diff
@@ -11,3 +11,9 @@ if (v[0] == 0 && v[1] < 12) {
                 process.version + " found");
   process.exit(1);
 }
+if (v[0] == 6 && v[1] == 2 && v[2] == 1) {
```

Comment:
You are of course right, I can't expect node to read the text of the warning message and perform the comparison based on its wording instead of the code I wrote…